packages/autorest.typescript/test/integration/generated/readmeFileChecker/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@msinternal/keyvault-secrets",
+  "name": "@azure/keyvault-secrets",
```

Comment:
This looks strange

Reply:
Guess this is also because the configuration in the readme.md has higher priority than the command line.